src/views/Layout.css

```diff
@@ -7,7 +7,7 @@
  *
  * @see: https://developer.mozilla.org/en-US/docs/Web/CSS/env
  */
-.Layout {
+/* .Layout {
```

Comment:
Seen this on other PRs, vast swaths of code and CSS that have been commented out. Just delete it, YAGNI